cmd/fleet/serve.go

```diff
@@ -840,7 +840,14 @@ the way that the Fleet server works.
 			launcher := launcher.New(svc, logger, grpc.NewServer(), healthCheckers)
 
 			rootMux := http.NewServeMux()
-			rootMux.Handle("/healthz", service.PrometheusMetricsHandler("healthz", health.Handler(httpLogger, healthCheckers)))
+			rootMux.Handle("/healthz", service.PrometheusMetricsHandler("healthz", health.Handler(httpLogger, healthCheckers, func() any {
```

Comment:
This is a workaround for the cyclic import issue when the `ds.GetHealthCheckCronStats()` or the struct is referenced in `health.go`. Not sure if this is the best way to go, open to suggestions.